IoTPiCar.py

```diff
@@ -19,20 +27,26 @@
 SERVO_ZERO_ANGLE = 0
 SERVO_MIN_ANGLE = -90
 SERVO_TIME = 0.5
+servo_currentAngle = SERVO_ZERO_ANGLE
+MAP_SIZE = 200
+envoriment_map = np.zeros((MAP_SIZE,MAP_SIZE), dtype='uint8')
 
-def isDetected(distances: list):
-    for distance in distances:
-        if 'found' in distance:
-            return True
-    return False
+#Considering the car starts from the bottom center
+car_start_map_x = MAP_SIZE/2 -1
+car_start_map_y = 0
 
-def detectionSubset(distances: list):
-    MAX_RANGE = 19
-    detection_list = []
-    for distance in distances:
-        if 'current-angle' in distance and abs(distance['current-angle']) < MAX_RANGE:
-            detection_list.append(distance)
-    return detection_list
+#Considering the car desitnation to be last mid point of the map
+car_destination_map_x = MAP_SIZE/2 -1
+car_destination_map_y = MAP_SIZE -1
+
+def set_servo_angle(angle: int):
+    global servo_currentAngle
+    servo_currentAngle = angle
+    servo.set_angle(angle)
+
+def get_servo_angle():
+    global servo_currentAngle
+    return(servo_currentAngle)
```

Comment:
we could just have the servo_currentAngle as a global variable in main, or elsewhere without sets. Please check.